Update WordPressKit to 14.0.1

crazytonyli opened the pull request:
Including a bug fix to getting feature announcement and zendesk metadata. See
https://github.com/wordpress-mobile/WordPressKit-iOS/pull/746 and https://github.com/wordpress-mobile/WordPressKit-iOS/pull/747.

See the full 14.0.0 - 14.0.1 changes [here][diff].

[diff]: https://github.com/wordpress-mobile/WordPressKit-iOS/compare/14.0.0...14.0.1

dangermattic commented:
<!--
  0 Errors
  1 Warning: This PR is assigned to the mil...
  0 Messages
  0 Markdowns
-->
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is assigned to the milestone <a href="https://github.com/wordpress-mobile/WordPress-iOS/milestone/272">24.4 ❄️</a>. The due date for this milestone has already passed.<br />
Please assign it to a milestone with a later deadline or check whether the release for this milestone has already been finished.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

crazytonyli committed:
Update WordPressKit to 14.0.1

Including a bug fix to getting feature announcement and zendesk
metadata. See
https://github.com/wordpress-mobile/WordPressKit-iOS/pull/746 and
https://github.com/wordpress-mobile/WordPressKit-iOS/pull/747.

See the full 14.0.0 - 14.0.1 changes [here][diff].

[diff]: https://github.com/wordpress-mobile/WordPressKit-iOS/compare/14.0.0...14.0.1

guarani reviewed (approved):
Thanks @crazytonyli, tested and working 👍

crazytonyli commented:
@mokagio, FYI, this PR is merged into `release/24.4`.
